app/views/exam_templates/_student_info.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= form_for [@assignment, exam_template],
+             url: add_fields_assignment_exam_template_path(id: exam_template.id) do |f| %>
+
+  <p>
+    <%= check_box_tag :automatic_parsing, true,
+                      exam_template.automatic_parsing,
+                      onchange: "toggle_cover_page(#{exam_template.id}, #{exam_template.cover_fields.split(',')}); return true;" %>
+    Automatically parse student information<br>
```

Comment:
Make all of the UI strings internationalized, by adding them to `config/locales/views/exam_templates/*.yml`.